docs/.gitignore

```diff
@@ -1,3 +1,3 @@
 build/
 site/
-src/generated/*
+#src/generated/*  # this should be removed
```

Comment:
why?

Reply:
I have a bunch of items that will be removed when the upgrade is complete

Reply:
the thing is this causes a couple of files to be commited that shouldn't be

Reply:
Yes I am aware ... have been holding back more edits, mostly doc related.